xml/containers-buildah-overview.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+    [
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter xml:id="cha-buildah-overview" xml:lang="en"
+    xmlns="http://docbook.org/ns/docbook" version="5.0"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&buildah; Overview</title>
+ <para>
+  <link xlink:href="https://buildah.io/">&buildah;</link> is tool for building
+  OCI-compliant container images. &buildah; can handle the following tasks.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Create containers from scratch, or from an existing image.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Create an image from a working container or via Dockerfile.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Build images in the OCI image format or the traditional upstream &deng;
+    image format.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Mount a working container's root filesystem for manipulation.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Use the updated contents of a container's root filesystem as a filesystem
+    layer to create a new image.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Delete a working container or an image and rename a local container.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  Compared to &deng;, &buildah; has several important advantages.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    &buildah; adheres the UNIX philosophy by focusing only on one task:
```

Comment:
Some would say that's a disadvantage... IMO it doesn't belong in the list here, more into the description of buildah